konfera/admin.py

```diff
@@ -321,3 +321,25 @@ class OrganizerAdmin(admin.ModelAdmin):
 
 
 admin.site.register(Organizer, OrganizerAdmin)
+
+
+class EmailTemplateAdmin(admin.ModelAdmin):
+    list_display = ('name', 'text_template', 'html_template', 'counter')
```

Comment:
we dont need to see html_template here, I am not even sure about the text_template how it would look it the text is too long (which in mail might be). So let stay with text_template only for now.